app/services/importers/doc/sentence.rb

```diff
@@ -16,8 +17,11 @@ def perform
         file.body.string.split("\r\n").each do |line|
           bar.increment!
           data = line.unpack(field_pattern).map(&:squish)
-          find_and_save_sentence(data)
+
+          @sentences << find_and_save_sentence(data)
         end
+        @sentences.compact!
+        ::Doc::Sentence.upsert_all(@sentences)
```

Comment:
`::Doc::Sentence.find_or_initialize_by(doc_profile_id: doc_mapping[data[0].to_i], sentence_id: data[1])`

it needs to be unique by doc_profile_id and sentence_id